New version: MadNLPHSL v0.6.0

JuliaRegistrator opened the pull request:
- Registering package: MadNLPHSL
- Repository: https://github.com/MadNLP/MadNLP.jl
- Created by: @amontoison
- Version: v0.6.0
- Commit: 12838df5cb31c5d509bd91fff4d457718442d14b
- Reviewed by: @amontoison
- Reference: https://github.com/MadNLP/MadNLP.jl/commit/12838df5cb31c5d509bd91fff4d457718442d14b#commitcomment-152379392
- Description: A solver for nonlinear programming with GPU support
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- breaking -- Require the official release 2024.11.28 or newer;
- Use the libraries `HSL_jll.libhsl_subset` and `HSL_jll.libhsl_subset_64` instead of `HSL_jll.libhsl`;
- Use the new generic wrappers of `HSL.jl` introduced in the release `0.5.0`.
`````
<!-- END RELEASE NOTES -->

<!-- 3ce55c06478276ba47412b3886bf0a3489a11e60c64636e26b2d5231439d333c47d2e5d749d024981da7521322dad986fbe3b49107ec744bc2e694750f1451d1b77088142a1e27255d4bfdbe5549658131522f199a2c4a6583d171c5a921a7c12589e88eacc769fe43a4f6981e8abb229a228c45bf318fa59ff65213c2a27d24c5bd7c2b11a449166613e5adcf2f6349af55b0533926d9a64b8a8807a463b2fe1159166edde520a7ed41de1c35eaf1beacf463a04effc1b175ef31e75eabae8b6dd915ba80e6fa804ef8bd6b2a65d4f3707e0e5bc687496b2b8986a50b5edcce -->

amontoison commented:
@JuliaRegistrator register

Release notes:

## Require the official release 2024.11.28 or newer
## Use the libraries `HSL_jll.libhsl_subset` and `HSL_jll.libhsl_subset_64` instead of `HSL_jll.libhsl`
## Use the new generic wrappers of `HSL.jl` introduced in the release `0.5.0`.

amontoison commented:
@JuliaRegistrator register

Release notes:

- Require the official release 2024.11.28 or newer;
- Use the libraries `HSL_jll.libhsl_subset` and `HSL_jll.libhsl_subset_64` instead of `HSL_jll.libhsl`;
- Use the new generic wrappers of `HSL.jl` introduced in the release `0.5.0`.

amontoison commented:
[noblock]

JuliaRegistrator committed:
New version: MadNLPHSL v0.6.0

UUID: 7fb6135f-58fe-4112-84ca-653cf5be0c77
Repo: https://github.com/MadNLP/MadNLP.jl.git
Tree: 4241cf4b0a8a4f96691f77738bc02a21ad8bd4bf

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
